pixeltable/io/globals.py

```diff
@@ -267,3 +273,90 @@ def import_json(
+    are stored in. Images that are not already on disk (such as frames extracted using a
+    [`FrameIterator`][pixeltable.iterators.FrameIterator]) will first be written to disk in the specified
+    `image_format`.
+
+    The label parameters accept one or more sets of labels of each type. If a single `Expr` is provided, then it will
+    be exported as a single set of labels with a default name such as `classifications`.
+    (The single set of labels may still containing multiple individual labels; see below.)
+    If a list of `Expr`s is provided, then each one will be exported as a separate set of labels with a default name
+    such as `classifications`, `classifications_1`, etc. If a dictionary of `Expr`s is provided, then each entry will
+    be exported as a set of labels with the specified name.
+
+    __Requirements:__
+
+    - `pip install fiftyone`
+
+    Args:
+        tbl: The table from which to export data.
+        images: A column or expression that contains the images to export.
+        image_format: The format to use when writing out images for export.
+        classifications: Optional image classification labels. If a single `Expr` is provided, it must be a table
+            column or an expression that evaluates to a list of dictionaries. Each dictionary in the list corresponds
+            to an image class and must have the following structure:
+
+            ```python
+            {'label': 'zebra', 'confidence': 0.325}
+            ```
+
+            If multiple `Expr`s are provided, each one must evaluate to a list of such dictionaries.
+        detections: Optional image detection labels. If a single `Expr` is provided, it must be a table column or an
+            expression that evaluates to a list of dictionaries. Each dictionary in the list corresponds to an image
+            detection, and must have the following structure:
+
+            ```python
+            {
+                'label': 'giraffe',
+                'confidence': 0.99,
+                'bounding_box': [0.081, 0.836, 0.202, 0.136]  # [x, y, w, h], fractional coordinates
+            }
+            ```
+
+            If multiple `Expr`s are provided, each one must evaluate to a list of such dictionaries.
+
+    Returns:
+        A Voxel51 dataset.
+
+    Example:
+        Export the images in the `image` column of the table `tbl` as a Voxel51 dataset, using classification
+        labels from `tbl.classifications`:
+
+        >>> export_as_fiftyone(
+        ...     tbl,
+        ...     tbl.image,
+        ...     classifications=tbl.classifications
+        ... )
+
+        See the [Working with Voxel51 in Pixeltable](https://docs.pixeltable.com/docs/working-with-voxel51) tutorial
+        for a fully worked example.
+    """
+    Env.get().require_package('fiftyone')
+
+    import fiftyone as fo
+    from pixeltable.io.fiftyone import PxtImageDatasetImporter
+
+    if not images.col_type.is_image_type():
```

Comment:
Additional checks:
- img expr should be bound by table (see Expr.is_bound_by())
- same for classification and detection exprs